rfc/0033-quilt-mappings-and-clean-room.md

```diff
@@ -0,0 +1,97 @@
+
+It's also clear that Mojang feels that their mappings should be used in this way - or at least have no wish to prevent it - given that they haven't intervened in other projects that use it in a similar way.
+
+It's become abundantly clear that Yarn's cleanroom isn't very clean, and that it doesn't serve any real purpose in today's modding ecosystem.
+
+### Changes to the Ecosystem
+
+As mentioned earlier, a lot has changed in the time since Yarn was concieved:
+
+* Most other modloader projects are using Mojmap, or something layered on top of it
+* MCP has been retroactively relicensed to the zlib license, and is no longer being maintained as an isolated mappings project
+* It's become clear that Microsoft has many other ways to kill off modloader projects, even though it doesn't have any discernable motivation to do so - and regardless of whether they'd be in the right to do so, it's unlikely that anyone would be able or willing to fund a court case in defense
+
+### Conclusion
+
+In conclusion, the current approach to Yarn doesn't really make sense given the modern modding ecosystem. While this RFC has no direct bearing on upstream Yarn, the Quilt Project feels that this RFC is the best way to move forward with Quilt Mappings.
+
+## Explanation
+
+Should this RFC be approved:
+
+* Quilt's fork of Yarn will be renamed to "Quilt Mappings" (QM)
+* All restrictions on referencing mappings projects (other than Quilt Mappings) within Quilt's community and development spaces will be lifted
+* A file named `ATTRIBUTION.md` will be created in the Quilt Mappings repository, which will properly attribute both the MCP project and Mojang for any names that may be inspired by or attributed to those projects, with the possibility of adding more as required
+* The Quilt Mappings project will provide additional attribution to other mappings projects and providers (including MCP and Mojang) as required, which may also include changes to Quilt's toolchain
+* The Quilt Mappings repository will have its current restrictions regarding mentions of other mappings projects lifted, and replaced with a requirement that that contributors may only reference other mappings projects that are public domain, licensed under the CC0, or attributed in the `ATTRIBUTION.md` file
+* The other Quilt Mappings contribution requirements, such as quality of names or conventions and standards will not be changed by this RFC
+* This will not affect Quilt's other plans going forward, and everything else will continue as normal - including the planned move from intermediary to Hashed Mojmap.
+
+These changes are likely to be somewhat gradual, but it's expected that they'll be made relatively quickly.
```

Comment:
"gradual but quick"? this feels like a bit of a contradiction

Reply:
Gradual just means "not all at once", really

Reply:
> These changes are likely to happen in many steps, but it's expected that they'll will be made in quick succession.

Reply:
@zml2008 is this a good clarification?